src/main/js/routes/layout/layout.js

```diff
@@ -112,8 +119,9 @@ function Layout() {
                         </div>
                         <div className="flex-1 h-screen p-6 overflow-x-auto w-full">
                             <div>
-                                <div className="space-x-1 w-full">
-                                    <div className="text-right font-flow"
+                                <div className="space-x-1 w-full parent-table">
+                                    <div className="text-left font-flow left-child">{`${(getHost())}`}</div>
```

Comment:
nit: there's not need for this to be a function, you could just compute the hostname above and React will take care of re-rendering when it changes

Reply:
Makes total sense. A lot cleaner.